examples/argocd/main.tf

```diff
@@ -1,6 +1,4 @@
-provider "aws" {
-  region = local.region
-}
+provider "aws" {}
```

Comment:
lets keep the region so that its known where the resources are deployed. without it, its up to the default fallbacks (i.e. - IAM profile, environment variables, etc.) as to where the resource are created